frameworks/elastic/build.gradle

```diff
@@ -22,7 +22,7 @@ ext {
 dependencies {
     compile project(":scheduler")
     compile project(":executor")
-    compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.21'
+    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
```

Comment:
Can we put this dependency in the `allProjects` or in `scheduler` and `executor` instead of every frameworks?